public/jsx/views/complete-payment.jsx

```diff
@@ -16,12 +19,26 @@ module.exports = React.createClass({
     };
   },
 
+  handleClick: function(e) {
+    e.preventDefault();
+    if (window.parent !== window) {
+      // Note: the targetOrigin is wide open.
+      // Nothing sensitive should be sent whilst
+      // that's the case.
+      console.log('Telling parent to close modal.');
+      window.parent.postMessage('close', '*');
```

Comment:
I'm 90% sure that we'll need to send more messages in the future (to indicate a finished purchase) so you might as well make this an object. I'd suggest: ``postMessage({event: 'window-closed'}, '*')`` or something. That way we could do ``{event: 'purchase-completed', data: {productId: ...}}``. Of course we'll probably change it later when we understand it better so doesn't matter, just a thought.